src/App/ComponentsDocumentation/components/Slab/index.js

```diff
@@ -1,210 +1,66 @@
+            <li>When using a slab component, the border radius could differ for different devices and sizes. Border radius always requires to be adjusted accordingly to the dimensions.</li>
+        </ul>
+        <h3>How to adjust slab radius</h3>
+        <div className="row">
+            <div className="col-lg-6 col-sm-12">
+                <div className="slab slab-plain slab-border-success pb-1">
+                    <h4 className="mb-3 mt-0">Do</h4>
+                    <div className="d-flex justify-content-center mt-3 mb-3">
+                        <img className="img-fluid" src={`${basename}img/documentation/slab/do-nested-slabs.png`} alt="A slab with a smaller slab inside of it. Their border radius values are equal, the padding size is therefore not consistent." />
                     </div>
                 </div>
-                <div className="col-6">
-                    <div className="slab slab-plain slab-border-error">
-                        <h3>Don&apos;t</h3>
-                        <ul className="list list-bullet">
-                            <li>Don&apos;t do this</li>
-                            <li>Don&apos;t do that</li>
-                            <li>Don&apos;t to the other</li>
-                        </ul>
+            </div>
+            <div className="col-lg-6 col-sm-12">
+                <div className="slab slab-plain slab-border-error pb-1">
+                    <h4>{"Don't"}</h4>
+                    <div className="d-flex justify-content-center mt-3 mb-3">
+                        <img className="img-fluid" src={`${basename}img/documentation/slab/dont-nested-slabs.png`} alt="A slab with a smaller slab inside of it. Their border radius value are not equal, which makes the padding consistent."/>
                     </div>
                 </div>
             </div>
-        </ComponentPreview>
-    </>
-);
-
-const SlabElevated = () => (
-    <>
-        <h2 id="elevated-slab">Elevated slab</h2>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <div className="slab slab-default slab-elevated">
-                Look ma, im elevated!
-            </div>
-        </ComponentPreview>
-    </>
-);
-
-const SlabWell = () => (
-    <>
-        <h2 id="slab-well">Slab well</h2>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <div className="slab slab-default slab-well">
-                Look ma, im in a well!
-            </div>
-        </ComponentPreview>
-    </>
+        </div>
+        <br/>
+        <div> When using a smaller slab in a slab component, in that case the inner slab needs to adjust its border radius accordingly in order the padding between the slabs to remain relevant and consistent.</div>
```

Comment:
Weird sentence.. Use this
"When using a smaller slab in a slab component, in that case, the inner slab needs to adjust its border radius accordingly in order for the padding between the slabs to remain relevant and consistent"